config/routes.rb

```diff
@@ -37,6 +37,7 @@
     post "/zendesk/exports", to: "zendesk_csv_exports#create"
     get "/zendesk/confirm-deletion", to: "zendesk#confirm_deletion"
     post "/zendesk/confirm-deletion", to: "zendesk#destroy"
+    get "/zendesk/performance", to: "zendesk_performance#index"
```

Comment:
I think that could work; maybe with slightly a different heading layout like:

```
# Zendesk

## Performance

Visit the [Zendesk performance dashboard] to see analytics of previously deleted tickets.

## Tickets to be deleted

The following ~etc~
```

Alternatively, instead of an inline link and separate page, we could add the content inline to the page, and use a tabbed layout [^0] to break it up:

```
[ Tickets to be deleted ] [ Previously deleted tickets ] [ Performance dashboard ]
```

Tabs might interfere with the pagination helpers...

[^0]: https://govuk-components.netlify.app/components/tabs/

Reply:
See my previous comment; `Zendesk` can be the main H1 of the whole page, then a `H2 Performance`, then another `H2 Tickets to be deleted` to prevent repeating the Zendesk thing over and over.